Watchman.Discord/Areas/Protection/Services/PunishmentsCachingService.cs

```diff
@@ -82,12 +105,17 @@ public async Task AddUserPunishment(ulong userId, Punishment punishment)
                     var userId = x.Key;
                     var punishments = x.Select(x =>
                     {
-                        var option = (PunishmentOption) (int) x.Option;
+                        var option = (PunishmentOption)(int)x.Option;
                         return new Punishment(option, x.GivenAt, x.Time);
                     });
                     return new KeyValuePair<ulong, IEnumerable<Punishment>>(userId, punishments);
                 })
                 .ToDictionary(x => x.Key, x => x.Value.ToList());
         }
+
+        private void AddWarnToDictionary(WarnEvent warnToAdd)
+        {
+            _warnsByServer[warnToAdd.ServerId][warnToAdd.ReceiverId].Add(warnToAdd);
```

Comment:
a czy przypadkiem nie jest tak, że jeśli nie znajdzie obiektu to wywali wyjątek?